web-app/src/components/BookingModal/BookingModalForm/index.js

```diff
@@ -52,6 +55,22 @@ const BookingModalForm = props => {
     }
   };
 
+  const remainingDays = (getTotalDays - getApprovedDays);
+  const pendingDays = getPendingDays;
+  const availableDays = (remainingDays - pendingDays);
+  const showRemainingHolsAlertMessage = () => {
+    return (remainingDays === 0 || bookingDuration > remainingDays) || (availableDays < bookingDuration);     
+  };
+
+  const holidayAlertMessage = () => {
+    return (
+      <HolidayAlert>
+        <tag>{availableDays} Days Available</tag>
```

Comment:
I may be silly here but I've never seen the <tag> element before and it's not on https://www.w3schools.com/tags/ does it exist, if so what purpose does it serve?